qa/rpc-tests/wallet_listreceived.py

```diff
@@ -42,10 +42,14 @@ def run_test_release(self, release, height):
 
         # Decrypted transaction details should be correct
         pt = self.nodes[1].z_viewtransaction(txid)
+
         assert_equal(pt['txid'], txid)
         assert_equal(len(pt['spends']), 0)
         assert_equal(len(pt['outputs']), 1 if release == 'sprout' else 2)
 
+        # Expect one internal output and one external.
```

Comment:
```suggestion
        # Expect one external output. (For non-Sprout, the other is internal.)
```

Reply:
No longer testing Sprout in the same place.